kafka-ui-api/src/main/java/com/provectus/kafka/ui/model/KafkaCluster.java

```diff
@@ -16,6 +16,7 @@ public class KafkaCluster {
   private final String name;
   private final String version;
   private final Integer jmxPort;
```

Comment:
I would suggest to rename:
1. Metrics
1.1 port
1.2 type: JMX/PrometheusEndpoint
1.3 basicAuth
1.3.1 username
1.3.2 password
1.4 ssl 
1.4.1 enabled true
1.4.2 truststore
1.4.3 keystore